cmake/setup_deal_ii.cmake

```diff
@@ -100,7 +102,11 @@ set(DEAL_II_VERSION ${DEAL_II_VERSION_MAJOR}.${DEAL_II_VERSION_MINOR}.${DEAL_II_
 #                                                                      #
 ########################################################################
 
-set(DEAL_II_PROJECT_CONFIG_NAME "${DEAL_II_PACKAGE_NAME}")
+string(REPLACE "." "" _namespace "${DEAL_II_PACKAGE_NAME}")
+string(TOLOWER "${_namespace}" _namespace)
+set_if_empty(DEAL_II_NAMESPACE "${_namespace}")
```

Comment:
Is there a good reason to not just use `dealii` for `DEAL_II_NAMESPACE`?

Reply:
The rest of the file derives all names from `${DEAL_II_PACKAGE_NAME}`, thus the small dance to derive "dealii" from it.